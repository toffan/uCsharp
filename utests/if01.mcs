int main() {
    if (1) {
        // nothing
    } else {
        // nothing
    }

    return 0;
}
