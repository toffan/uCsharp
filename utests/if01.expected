JUMP __main
_main:
    LOADL 1
    JUMPIF (0) _X0:
    JUMP _X1:
    _X0:
    _X1:

    LOADL 0
    RETURN (1) 0
    RETURN (0) 0
__main:
    CALL (SB) _main
    HALT
